modules/installation-configuration-parameters.adoc

```diff
@@ -1166,6 +1196,61 @@ Additional IBM Cloud VPC configuration parameters are described in the following
 --
 endif::ibm-cloud[]
 
+ifdef::ibm-power-vs[]
+[id="installation-configuration-parameters-additional-ibm-cloud_{context}"]
+== Additional IBM Power VS configuration parameters
+
+Additional IBM Power VS configuration parameters are described in the following table:
+
+.Additional IBM Power VS parameters
+[cols=".^1,.^6a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.powervs.resourceGroupName`
+|The name of an existing resource group to install your cluster to. This resource group must only be used for this specific cluster because the cluster components assume ownership of all of the resources in the resource group.
+|String, for example `existing_resource_group`.
+
+//|`platform.ibmcloud.dedicatedHosts.profile`
+//|The new dedicated host to create. If you specify a value for `platform.ibmcloud.dedicatedHosts.name`, this parameter is not required.
+//|Valid IBM Cloud VPC dedicated host profile, such as `cx2-host-152x304`. [^2^]
+
+//|`platform.ibmcloud.dedicatedHosts.name`
+//|An existing dedicated host. If you specify a value for `platform.ibmcloud.dedicatedHosts.profile`, this parameter is not required.
+//|String, for example `my-dedicated-host-name`.
+
+//|`platform.ibmcloud.type`
+//|The instance type for all IBM Cloud VPC machines.
+//|Valid IBM Cloud VPC instance type, such as `bx2-8x32`. [^2^]
+
+|`platform.powervs.memoryGiB`
+|The size of a virtual machine's memory, in GB.
+|The valid integer must be an integer number of GB that is at least 2 and no more than 64, depending on the machine type.
+
+|`platform.powervs.procType`
+|The ProcType defines the processor sharing model for the instance.
+|The valid values are Capped, Dedicated and Shared.
+
+|`platform.powervs.processors`
+|The Processors defines the processing units for the instance.
+|The number of processors must be from .5 to 32 cores. The processors must be in increments of .25.
```

Comment:
@bkhadars could you please confirm the maximum limit mentioned here ? thanks !

Reply:
yes, these are the current max and minimum limits

Reply:
thanks @bkhadars !